test/envs/error.yaml

```diff
@@ -12,5 +12,7 @@ profiles:
     workers:
       - addons:
           - name: error
+            args: ["10"]
       - addons:
-          - name: example
+          - name: sleep
+            args: ["60"]
```

Comment:
Need to add another cluster to test that failures in one cluster executor abort all executors.